examples/connext_dds/dynamic_data_skip_serialization/c++11/README.md

```diff
@@ -0,0 +1,69 @@
+# Example Code: Skip Data Serialization for Data Recording
```

Comment:
```suggestion
# Example Code: Skip Data Deserialization for Data Recording
```

Reply:
see my other comment as to why I'd prefer to keep "Serialization"